### PR TITLE
[CDAP-17447] Emit bytes written metric for database sinks

### PR DESCRIPTION
**Pull request by rmstar**

### CODE REVIEW

File: database-plugins/src/main/java/io/cdap/plugin/DBRecord.java [hunk 1]
```diff
@@ -72,6 +73,7 @@
   public DBRecord(StructuredRecord record, int [] columnTypes) {
     this.record = record;
     this.columnTypes = columnTypes;
+    this.bytesWritten = 0;
```
Comment (chtyim) [inline database-plugins/src/main/java/io/cdap/plugin/DBRecord.java]: No need. Default value of long field is already 0L.

File: database-plugins/src/main/java/io/cdap/plugin/DBRecord.java [hunk 2]
```diff
@@ -240,14 +249,20 @@ private void writeToDB(PreparedStatement stmt, Schema.Field field, int fieldInde
       switch (fieldLogicalType) {
         case DATE:
           stmt.setDate(sqlIndex, Date.valueOf(record.getDate(fieldName)));
+          bytesWritten += ((Long) record.get(fieldName)).BYTES;
```
Comment (chtyim) [inline database-plugins/src/main/java/io/cdap/plugin/DBRecord.java]: Long.BYTES is a static constant, shouldn't be accessing it via instance.

File: database-plugins/src/main/java/io/cdap/plugin/DBRecord.java (hunk 1 above)
Comment (rmstar) [reply database-plugins/src/main/java/io/cdap/plugin/DBRecord.java]: removed.

File: database-plugins/src/main/java/io/cdap/plugin/DBRecord.java (hunk 2 above)
Comment (rmstar) [reply database-plugins/src/main/java/io/cdap/plugin/DBRecord.java]: fixed

**Review by chtyim — APPROVED** [review]
LGTM

**Pull request merged**